command/test-fixtures/hcl/reprepare/shell-local.pkr.hcl

```diff
@@ -0,0 +1,13 @@
+source "null" "example" {
+    communicator = "none"
+}
+
+build {
+	sources = [
+		"source.null.example"
+	]
+	provisioner "shell-local" {
+		script = "./test-fixtures/hcl/reprepare/hello.sh"
```

Comment:
this path is not working for Windows, I need to allow HCL2 templates to use `path` variables in order to fix this on windows. I'll open another PR.

Reply:
Okay, I just opened #9628 to allow this; I'm going to cherry-pick it here and fix that issue.